linker/shared/src/main/scala/org/scalajs/linker/standard/StandardLinkerImpl.scala

```diff
@@ -16,15 +16,16 @@ import java.util.concurrent.atomic.AtomicBoolean
 import org.scalajs.logging.Logger
 import org.scalajs.io._
 
-import org.scalajs.linker.standard._
+import org.scalajs.linker._
 import org.scalajs.linker.analyzer.SymbolRequirement
 import org.scalajs.linker.frontend.LinkerFrontend
 import org.scalajs.linker.frontend.optimizer.IncOptimizer
 import org.scalajs.linker.backend.{LinkerBackend, BasicLinkerBackend}
 import org.scalajs.linker.irio._
 
-/** The Scala.js linker */
-final class Linker private (frontend: LinkerFrontend, backend: LinkerBackend)
+/** Standard implementation of a Scala.js linker. */
+private[standard] final class StandardLinkerImpl private (
+    frontend: LinkerFrontend, backend: LinkerBackend)
```

Comment:
Can't you make this `private`? The companion should still see it, no?

Reply:
Indeed. Probably came from a transient state.